eval.c

```diff
@@ -1346,9 +1346,19 @@ rb_using_module(const rb_cref_t *cref, VALUE module)
 
 /*
  *  call-seq:
- *     target    -> class
+ *     target    -> class_or_module
  *
  *  Return the class or module refined by the receiver.
+ *
+ *     module M
+ *       refine String do
+ *         def indent(level)
```

Comment:
Best remove the method definition here, it doesn't affect the result and is unrelated to the method being documented.